Modules/FixDaDatabase.md

```diff
@@ -0,0 +1,16 @@
+# Name: FixDaDatabase
+
+## Description:
+This module has been created to remediate a common problem with Direct Access servers. It can determine if an Index is present for the Direct Access is present. And it can either remove and create that index.
+
+| Works with | Author(s) | Project URL(s) | Creation Date | Tags |
+|------------|--------|-------------------|---------------|------|
+| DirectAccess 2012 (R2) | Jaap Brasser ([b](https://www.jaapbrasser.com) \|  [t](https://twitter.com/Jaap_Brasser)) | [GitHub](https://github.com/jaapbrasser/FixDaDatabase) | Apr 2016 | PowerShell, DirectAccess, Database |
+____
+### Shared blog posts:
+Jaap Brasser ([b](https://www.jaapbrasser.com) \|  [t](https://twitter.com/Jaap_Brasser))
+- [Fix DirectAccess Windows Internal Database – PowerShell Module](https://www.jaapbrasser.com/fix-directaccess-windows-internal-database-powershell-module/)
+____
+### Extended documentation:
+Jaap Brasser ([b](https://www.jaapbrasser.com) \|  [t](https://twitter.com/Jaap_Brasser))
+- [FixDaDatabase - ReadTheDocs.io](http://fixdadatabase.readthedocs.io/en/latest/)
```

Comment:
Please add this link on the "Project URL(s)" cell up there with 
[ReadTheDocs](http://fixdadatabase.readthedocs.io/en/latest/)